### PR TITLE
Updated calotrigger table amd implement for EMCAL

### PR DESCRIPTION
**Pull request by mfasDa**
- Updated calotrigger table
- Adapt EMCAL trigger table to STU format in run3

### CODE REVIEW

File: RUN3/AliAnalysisTaskAO2Dconverter.cxx [hunk 1]
```diff
@@ -1904,39 +1906,96 @@ void AliAnalysisTaskAO2Dconverter::FillEventInTF()
...
+  // - For regular events store non-0 L1 ADCs of the 3 leading Gamma patches
   AliEMCALGeometry *geo = AliEMCALGeometry::GetInstanceFromRunNumber(fVEvent->GetRunNumber()); // Needed for EMCAL trigger mapping
   AliVCaloTrigger *calotriggers = fVEvent->GetCaloTrigger("EMCAL");
+  Bool_t fullPayload = gRandom->Uniform() < fFractionL1MonitorEventsEMCAL;
+  calotrigger.fIndexBCs = fBCCount;
+  calotrigger.fFastOrAbsID = 10001;
+  calotrigger.fLnAmplitude = fullPayload ? 1 : 0;
+  calotrigger.fTriggerBits = 0;
+  calotrigger.fCaloType = 1;
+  FillTree(kCaloTrigger);
```
Comment (jgrosseo) [inline RUN3/AliAnalysisTaskAO2Dconverter.cxx:1921]: I think you need to increase ncalotriggers_filled on the FillTree calls

Comment (mfasDa) [reply RUN3/AliAnalysisTaskAO2Dconverter.cxx:1921]: It is done below. The first 3 entries are global event properties which are mapped with unique FastOR Abs IDs to be interpreted as status codes. Consequently I start ncalotriggers_filled with 3.

**Comment by jgrosseo** [comment]
Waiting for comment from PHOS until the end of the day. Then merging this.

**Comment by jgrosseo** [comment]
+test

**Comment by jgrosseo** [comment]
+1

**Pull request merged**